src/components/MoreInfoWrapper/MoreInfo/MoreInfo.jsx

```diff
@@ -53,6 +53,38 @@ const MoreInfo = (props) => {
               attributes are not associated with privacy risks. They will be kept unmodified.
             </li>
           </ul>
+          <p><b>Prosecutor model</b></p>
+          <ul>
+            <li>
+              <p>
+              In the prosecutor model the attacker targets a specific individual
+                <br />
+              and it is assumed that the attacker already knows that data about the individual is contained in the dataset.
```

Comment:
It is assumed that the attacker has prior knowledge that the individual is within the dataset.